test/liquidate/reactivate.ts

```diff
@@ -61,7 +61,7 @@ describe('Reactivate Tests', () => {
     await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).reactivate(updatedCluster.operatorIds, minDepositAmount, updatedCluster.cluster)).to.emit(ssvNetworkContract, 'ClusterReactivated');
   });
 
-  it('Reactivate with 0 deposit and no validators emits ClusterReactivated', async () => {
+  it.skip('Reactivate with 0 deposit and no validators emits ClusterReactivated', async () => {
```

Comment:
@andrew-blox I think reactivating with 0 balance no longer makes sense, as the check for `isLiquidatable` will return true. Can a cluster with zero balance be reactivated?

Reply:
not sure, i mean if there are no validators in the cluster then i guess theoretically you can reactivate with 0 but no other way, even if you uahve 4 operators with 0 fee there will still be network foo